Config.uk

```diff
@@ -45,6 +45,15 @@ config LWIP_UNIKRAFT21X
 endchoice
 
 menu "Netif drivers"
+config LWIP_LOOPIF
+	bool "Loop interface"
+	default n
+	implies LWIP_LOOPBACK
+	help
+		A driver for localhost (e.g., 127.0.0.1) networking.
```

Comment:
`(e.g., 127.0.0.1)` -> `(e.g. 127.0.0.1)`. I think the `,` should be removed.

Reply:
The comma after `e.g.` or `i.e.` is - afaik - US-style English

Reply:
I just found this: https://en.wiktionary.org/wiki/e.g.